templates/html/extsearch.js

```diff
@@ -78,16 +78,60 @@ function searchFor(query,page,count) {
       $.each(data.items, function(i,item){
         let prefix = tagMap[item.tag];
         if (prefix) prefix+='/'; else prefix='';
+
```

Comment:
Please check whether the list is complete, (see also the doc/extsearch.doc and src/searchindex.cpp), current list is (updated February 16, 2024):
```
          source, function, slot,
          signal, variable, typedef, enum, enumvalue, property, event, related,
          friend, define, file, namespace, concept, group, package, page, dir, module,
          constants, library, type, union, interface, protocol category, exception, class,
          struct, service, singleton
```